src/cli.ts

```diff
@@ -131,15 +132,18 @@ async function performFirstPackAndCopyToCallForGivenArguments(
 
   await executePackageTargetAndCopyToDestinationDirectoriesCalls(excludedDestinationPaths, logger, packAndCopyCalls);
 
+  const yarnrcYmlExistence: Promise<boolean> = doesYarnrcYmlFileExist();
   await maybeOutputNextStepsText(
     maybeDestinationDirectoryToAddDependencyOn,
     rootTargets,
     outputPostCommandMessages,
     logger,
     process.cwd(),
-    doesPackageLockFileExist,
-    doesYarnLockFileExist,
-    () => tryGetProjectPackageJsonContent(path.resolve('./'))
+    doesPackageLockFileExist(),
+    doesYarnLockFileExist(),
+    yarnrcYmlExistence,
```

Comment:
I changed some of the async stuff by the principle of "start the work as soon as possible, block (await) on it as late as possible".